.github/workflows/cmake.yml

```diff
@@ -34,17 +47,29 @@ jobs:
         sudo apt-get install clang-format-11 libstdc++-10-dev libmpfr-dev libmpc-dev ninja-build npm
       # If we ever do any qemu-emulation on Github Actions, we should re-enable this:
       # e2fsprogs qemu-system-i386 qemu-utils
-    - name: Install prettier
-      run: sudo npm install -g prettier
+      if: ${{ runner.os == 'Linux' }}
     - name: Use GCC 10 instead
       run: sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 60 --slave /usr/bin/g++ g++ /usr/bin/g++-10
+      if: ${{ runner.os == 'Linux' }}
+
+    - name: Install MacOS dependencies
+      run: brew install coreutils ninja
+      if: ${{ runner.os == 'macOS' }}
+
+    - name: Install js dependencies
+      run: sudo npm install -g prettier
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 requests
     - name: Check versions
-      run: set +e; g++ --version; g++-10 --version; clang-format --version; clang-format-11 --version; prettier --version; python --version; python3 --version; ninja --version
+      run: set +e; g++ --version; g++-10 --version; clang-format-11 --version; prettier --version; python --version; python3 --version; ninja --version; flake8 --version
```

Comment:
Some script tries to determine on its own which `clang-format` it should use, so I'd prefer to log the versions of all binaries in each run. Note that a missing binary does not cause an error, that's what the `set +e` dooes. Suggestion: Keep the `clang-format --version`.

Reply:
put the clang-format --version back. even though it's not a real thing :)